lib/floe/workflow/choice_rule/data.rb

```diff
@@ -6,12 +6,14 @@ class ChoiceRule
       class Data < Floe::Workflow::ChoiceRule
         TYPES      = ["String", "Numeric", "Boolean", "Timestamp", "Present", "Null"].freeze
         COMPARES   = ["Equals", "LessThan", "GreaterThan", "LessThanEquals", "GreaterThanEquals", "Matches"].freeze
+        OPERATIONS = TYPES.each_with_object({}) { |dt, a| a[dt] = :"is_#{dt.downcase}?" } \
+                          .merge(COMPARES.each_with_object({}) { |op, a| a[op] = :"op_#{op.downcase}?" }).freeze
         # e.g.: (Is)(String), (Is)(Present)
         TYPE_CHECK = /^(Is)(#{TYPES.join("|")})$/
         # e.g.: (String)(LessThan)(Path), (Numeric)(GreaterThanEquals)()
         OPERATION  = /^(#{(TYPES - %w[Null Present]).join("|")})(#{COMPARES.join("|")})(Path)?$/
 
-        attr_reader :variable, :compare_key, :operation, :type, :compare_predicate, :path
+        attr_reader :variable, :compare_key, :operator, :type, :compare_predicate, :path
```

Comment:
This is changing the "public" api - not a big deal if it's internal, but is anything else using this?

Reply:
No it's fine, I just wanted to make sure that was accounted for or not.